src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketApi.java

```diff
@@ -26,6 +26,8 @@
 import com.cloudbees.jenkins.plugins.bitbucket.client.repository.UserRoleInRepository;
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
+
+import javax.annotation.Nullable;
```

Comment:
Please use the annotations that are legally compatible with redistribution in a docker image that includes an oracle JRE (i.e. The edu.umd.cs.findbugs one)